### PR TITLE
Bug 1135143 - Link to developer app results from "More Information" block

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Added link to developer's name. In line 210

### CODE REVIEW

File: src/templates/app/index.html [hunk 1]
```diff
@@ -207,7 +207,7 @@
         <div class="app-more-metadata">
           <dl>
             <dt>{{ _('Developer') }}</dt>
-            <dd>{{ this.author }}</dd>
+            <dd><a href="{{ url('search')|urlparams(src='this.author') }}">{{ this.author }}</a></dd>
```
Comment (diox) [inline src/templates/app/index.html]: You're almost there. However with this you are setting the `src` parameter, when you want to set the `author` parameter. So you need to adjust the `urlparams` call.

Comment (atiqueahmedziad) [inline src/templates/app/index.html]: will i write this ? 
>  ``<dd><a href="{{ url('search')|urlparams(author='this.author') }}">{{ this.author }}</a></dd>``

File: src/templates/app/index.html [hunk 2]
```diff
@@ -207,7 +207,7 @@
         <div class="app-more-metadata">
           <dl>
             <dt>{{ _('Developer') }}</dt>
-            <dd>{{ this.author }}</dd>
+            <dd><a href="{{ url('search')|urlparams(author='this.author') }}">{{ this.author }}</a></dd>
```
Comment (ngokevin) [inline src/templates/app/index.html]: You accidentally added quotes around ```this.author```.

Comment (atiqueahmedziad) [inline src/templates/app/index.html]: Thanks... Updated :)

**Comment by andymckay** [comment]
r+ from @diox

**Comment by andymckay** [comment]
Any chance you could clear up the commit description here: https://github.com/atiqueahmedziad/fireplace/commit/4f7fe5f4ea975348b789fbb600a436cbb565e6db

**Comment by ngokevin** [comment]
sorry, kept getting distracted from applying this locally. looks good to me too besides the commit description. after this lands, you can write a test for it too if you want :)

**Comment by ngokevin** [comment]
Thanks!

**Comment by atiqueahmedziad** [comment]
@ngokevin @andymckay I am very much sorry for not replying & edited the commit as I was outside. Thanks to Kevin :)

**Pull request merged**